createTest.js

```diff
@@ -0,0 +1,39 @@
+const { Client } = require('pg');
+const client = new Client({
+  user: 'postgres',
+  password: 'Alvaro334',
+  // host: 'Ojumo',
+  port: 5432,
+  database: 'test'
+});
+execute();
+async function execute() {
+  try {
+    await client.connect();
+    console.log('Connected successfully');
+    await client.query('BEGIN');
+    await client.query(queryText1);
+    await client.query(queryText);
+    console.log('table created');
+
+    await client.query('COMMIT');
+  } catch (error) {
+    console.log('Failed on Error');
+    console.log(error);
+
+    await client.query('ROLLBACK');
+  } finally {
+    await client.end();
+    console.log('Cleaned');
+  }
+}
+const queryText1 = 'DROP TABLE IF EXISTS mydata';
+const queryText = ` CREATE TABLE IF NOT EXISTS
```

Comment:
I think you should declare these variables before the execute function(for the sake of readability). Also, I think the name of this variable should be something like `dropDBSQL` and `createDBSQL`( or something more descriptive).